eq_cir_converter_service/routers/schema_router.py

```diff
@@ -44,13 +48,11 @@ async def post_schema(
     logger.debug("Received current version %s and target version %s", current_version, target_version)
     logger.debug("Received schema: %s", schema)
 
-    """Validate the current and target version."""
-
+    logger.info("Validating the current and target version...")
```

Comment:
Why does this not get flagged by `EM` flake8-errmsg linter? Which linters should be enabled?

Reply:
Talked about this with @farres1 and he will enable relevant linters.